fs/copy.go

```diff
@@ -39,6 +39,7 @@ type XAttrErrorHandler func(dst, src, xattrKey string, err error) error
 
 type copyDirOpts struct {
 	xeh XAttrErrorHandler
+	xex map[string]bool
```

Comment:
nit: I'd likely have a longer variable name here like `xexclude` or `excludedXattrs` or at least an explanatory comment.  I'd also probably use a `map[string]struct{}` instead of a `map[string]bool`.

```suggestion
	// xex contains a set of xattrs to exclude when copying
	xex map[string]struct{}
```